Readme.md

```diff
@@ -47,3 +47,19 @@ Where:
 - `token` - the token to access the Qase API
 - `testrun-id` - the id of the test run in Qase
 - `output` - the file to save the expected results
+
+## How to use github action
+
+To use this tool in the github action, you need to create a workflow file in the `.github/workflows` directory and add
+the following step. For example:
+
+```yaml
+- name: Run Reporter Validator
+  uses: qase-tms/reporters-validator@v1
+  id: run_validator
+  with:
+    project-code: ${{ secrets.QASE_PROJECT_CODE }}
```

Comment:
I doubt that project code should be a secret, because it's not unique globally. But I don't insist ))